js/apps/account-ui/test/AdminClient.ts

```diff
@@ -0,0 +1,35 @@
+import KeycloakAdminClient from "@keycloak/keycloak-admin-client";
+import RealmRepresentation from "@keycloak/keycloak-admin-client/lib/defs/realmRepresentation";
+
+class AdminClient {
```

Comment:
I don't think we need to have a class for this, we can keep the instance in the module itself.

Reply:
Did the refactor work for this one.